mousewheel.jquery.json

```diff
@@ -22,6 +22,6 @@
     "homepage": "https://github.com/brandonaaron/jquery-mousewheel",
     "download": "https://github.com/brandonaaron/jquery-mousewheel/tags",
     "dependencies": {
-        "jquery": ">=1.2.2"
+        "jquery": ">=1.7.0"
```

Comment:
Maybe we should just remove this file since we have no plans to resurrect our plugins repo in its current form?